File: Backend/src/main/java/undersociety/models/Statistics.java
package undersociety.models;

public class Statistics {

	private double electronics = 0;
	private double furnitures = 0;
	private double appliances = 0;
	private double books = 0;
	private double clothes = 0;
	private double instock = 0;
	private double sold = 0;
	private double reserved = 0;
	private double totalUsers = 0;
	private double users = 0;
	private double admins = 0;
	private int earnings = 0;
	private int products = 0;
	private int posts = 0;
	private int customers = 0;
	private int companies = 0;
	
	public Statistics(double electronics, double furnitures, double appliances, double books, double clothes,
			double instock, double sold, double reserved, double totalUsers, double users, double admins, int earnings,
			int products, int posts, int customers, int companies) {
		super();
		this.electronics = electronics;
		this.furnitures = furnitures;
		this.appliances = appliances;
		this.books = books;
		this.clothes = clothes;
		this.instock = instock;
		this.sold = sold;
		this.reserved = reserved;
		this.totalUsers = totalUsers;
		this.users = users;
		this.admins = admins;
		this.earnings = earnings;
		this.products = products;
		this.posts = posts;
		this.customers = customers;
		this.companies = companies;
	}

	public double getElectronics() {
		return electronics;
	}

	public void setElectronics(double electronics) {
		this.electronics = electronics;
	}

	public double getFurnitures() {
		return furnitures;
	}

	public void setFurnitures(double furnitures) {
		this.furnitures = furnitures;
	}

	public double getAppliances() {
		return appliances;
	}

	public void setAppliances(double appliances) {
		this.appliances = appliances;
	}

	public double getBooks() {
		return books;
	}

	public void setBooks(double books) {
		this.books = books;
	}

	public double getClothes() {
		return clothes;
	}

	public void setClothes(double clothes) {
		this.clothes = clothes;
	}

	public double getInstock() {
		return instock;
	}

	public void setInstock(double instock) {
		this.instock = instock;
	}

	public double getSold() {
		return sold;
	}

	public void setSold(double sold) {
		this.sold = sold;
	}

	public double getReserved() {
		return reserved;
	}

	public void setReserved(double reserved) {
		this.reserved = reserved;
	}

	public double getTotalUsers() {
		return totalUsers;
	}

	public void setTotalUsers(double totalUsers) {
		this.totalUsers = totalUsers;
	}

	public double getUsers() {
		return users;
	}

	public void setUsers(double users) {
		this.users = users;
	}

	public double getAdmins() {
		return admins;
	}

	public void setAdmins(double admins) {
		this.admins = admins;
	}

	public int getEarnings() {
		return earnings;
	}

	public void setEarnings(int earnings) {
		this.earnings = earnings;
	}

	public int getProducts() {
		return products;
	}

	public void setProducts(int products) {
		this.products = products;
	}

	public int getPosts() {
		return posts;
	}

	public void setPosts(int posts) {
		this.posts = posts;
	}

	public int getCustomers() {
		return customers;
	}

	public void setCustomers(int customers) {
		this.customers = customers;
	}

	public int getCompanies() {
		return companies;
	}

	public void setCompanies(int companies) {
		this.companies = companies;
	}
	
}

File: Frontend/src/app/Class/Statistics/statistics.ts
export class Statistics {

    private electronics : Number;
	private furnitures : Number;
	private appliances : Number;
	private books : Number;
	private clothes : Number;
	private instock : Number;
	private sold : Number;
	private reserved : Number;
	private totalUsers : Number;
	private users : Number;
	private admins : Number;
	private earnings : Number;
	private products : Number;
	private posts : Number;
	private customers : Number;
	private companies : Number;

	constructor( electronics: Number,  furnitures: Number,  appliances: Number,  books: Number,  clothes: Number,  instock: Number,  sold: Number,  reserved: Number,  totalUsers: Number,  users: Number,  admins: Number,  earnings: Number,  products: Number,  posts: Number,  customers: Number,  companies: Number) {
		this.electronics =  electronics;
		this.furnitures =  furnitures;
		this.appliances =  appliances;
		this.books =  books;
		this.clothes =  clothes;
		this.instock =  instock;
		this.sold =  sold;
		this.reserved =  reserved;
		this.totalUsers =  totalUsers;
		this.users =  users;
		this.admins =  admins;
		this.earnings =  earnings;
		this.products =  products;
		this.posts =  posts;
		this.customers =  customers;
		this.companies =  companies;
	}


	public getElectronics(): Number {
		return this.electronics;
	}

	public getFurnitures(): Number {
		return this.furnitures;
	}

	public getAppliances(): Number {
		return this.appliances;
	}

	public getBooks(): Number {
		return this.books;
	}

	public getClothes(): Number {
		return this.clothes;
	}

	public getInstock(): Number {
		return this.instock;
	}

	public getSold(): Number {
		return this.sold;
	}

	public getReserved(): Number {
		return this.reserved;
	}

	public getTotalUsers(): Number {
		return this.totalUsers;
	}
   
	public getUsers(): Number {
		return this.users;
	}

	public getAdmins(): Number {
		return this.admins;
	}

	public getEarnings(): Number {
		return this.earnings;
	}

	public getProducts(): Number {
		return this.products;
	}

	public getPosts(): Number {
		return this.posts;
	}

	public getCustomers(): Number {
		return this.customers;
	}

	public getCompanies(): Number {
		return this.companies;
	}
}
